package.json

```diff
@@ -55,7 +57,10 @@
     "@eslint/eslintrc": "^2.0.2",
     "@eslint/js": "^8.37.0",
     "@release-it/conventional-changelog": "^4.3.0",
-    "@typescript-eslint/parser": "^5.36.2",
+    "@types/eslint": "^8.56.2",
+    "@types/estree": "^1.0.5",
+    "@typescript-eslint/parser": "^6.20.0",
+    "@typescript-eslint/rule-tester": "^6.20.0",
```

Comment:
https://github.com/eslint-community/eslint-plugin-eslint-plugin/actions/runs/7789482239/job/21241205874?pr=433:

```plaintext
> nyc --all --check-coverage --include lib mocha tests --recursive


Error: Cannot find module 'eslint/use-at-your-own-risk'
Require stack:
- /home/runner/work/eslint-plugin-eslint-plugin/eslint-plugin-eslint-plugin/node_modules/@typescript-eslint/rule-tester/dist/utils/config-validator.js
```

Is this blocked on dropping ESLint 7 support then?